components/PackageReadMePlaceholder.tsx

```diff
@@ -0,0 +1,53 @@
+import Html from './Html';
+
+type Props = {
+    data: {
+        packageName: string,
+        version: string,
+        title: string,
+        description: string
+    }
+}
```

Comment:
You can directly expose the fields as props:
```suggestion
type Props = {
        packageName: string,
        version: string,
        title: string,
        description: string
}
```